### ISSUE TITLE
Helper for contract verification

### ISSUE THREAD

**Issue by xermicus** [xermicus/revive#27]
For block explorers it would be beneficial to have a `verify` feature akin the one in [cargo contract ](https://github.com/use-ink/cargo-contract?tab=readme-ov-file#cargo-contract-verify).

**Comment by xermicus** [comment]
Yeah I agree, that could be even easier. The compilation artifacts should be deterministic anyways. @sourabhniyogi suggested to me that block explorers want a docker container and it wouldn't be a big issue - however let's wait and see what they actually need 👍

**Comment by athei** [comment]
I guess eventually we want to provide both. Ideally, both releases produce the same output. I remember that this wasn't the case for rustc, though.

**Comment by xermicus** [comment]
Depends on #93

**Comment by xermicus** [comment]
`resolc` shouldn't cause indeterministic builds without debug information (default) and without parallelism (default in Wasm)

To close this issue:
- [ ] Confirm that our parallelism (`parallel` feature flag) can't cause indeterministic builds or disable it alltogether and add a (non-default) indeterministic mode which enables parallelism again.
- [ ] Figure out what exactly causes indeterministic builds in LLVM 18 (if anything). Make sure to avoid them.
